resources/users.scss

```diff
@@ -117,6 +117,7 @@ tr:target {
 img.user-gravatar {
     display: block;
     border-radius: 6px;
```

Comment:
We should also standardize this. It looks like we're using 2px in comments but 6px in the users page.

Reply:
We're only using 2px in comments for small-width screens since we use a smaller gravatar size (otherwise the gravatar is a circle).